### PR TITLE
Set up testing and create snapshot tests for existing components

### PR DESCRIPTION
**Pull request by annekemcgrady**

### CODE REVIEW

**Comment by annekemcgrady** [comment]
@hndfaw  -  we can take a look at this together, I don't think conflicts are anything major but no need to merge yet.

**Comment by hndfaw** [comment]
@annekemcgrady thanks for pushing tests! I don’t think the conflict was a big deal, I resolved it. hopefully, all will be good.

File: package.json [hunk 1]
```diff
@@ -28,5 +28,15 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
```
Comment (letakeane) [inline package.json]: comment

File: src/App.test.js [hunk 2]
```diff
@@ -1,9 +1,28 @@
...
 import App from './App';
 
-it('renders without crashing', () => {
-  const div = document.createElement('div');
-  ReactDOM.render(<App />, div);
-  ReactDOM.unmountComponentAtNode(div);
+// it('renders without crashing', () => {
+//   const div = document.createElement('div');
+//   ReactDOM.render(<App />, div);
+//   ReactDOM.unmountComponentAtNode(div);
```
Comment (letakeane) [inline src/App.test.js]: question

File: src/components/Aside/Aside.test.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import Aside from './Aside.js';
+
+describe('Aside', ()=> {
```
Comment (letakeane) [inline src/components/Aside/Aside.test.js]: blah blah blah

File: package.json (hunk 1 above)
Comment (letakeane) [reply package.json]: explanation!

**Pull request merged**